h/templates/panels/navbar.html.jinja2

```diff
@@ -15,6 +15,13 @@
         {{ svg_icon('search', 'search-bar__icon') }}
         <div class="search-bar__lozenges" data-ref="searchBarLozenges"></div>
         <input class="search-bar__input" data-ref="searchBarInput" name="q" value="{{ q }}" placeholder="Search…" autocomplete="off">
+
+        {#- This seemingly pointless <input> is actually needed to make the
```

Comment:
The empty line before this comment will also be stripped from the HTML by Jinja2